Cargo.toml

```diff
@@ -19,6 +19,7 @@ bitflags = "1.2"
 vcell = "0.1"
 static_assertions = "1.1"
 fugit = "0.3.5"
+fixed = "1.28.0"
```

Comment:
`fixed` seems to require rustc 1.79 or newer. What do you think about putting cordic and `fixed` behind a feature?

Reply:
Ah yep, I was thinking about that. Sounds good!